esic_fe/src/components/Navbar/Navbar.jsx

```diff
@@ -1,12 +1,57 @@
-import React from "react";
+import React, { useState } from "react";
 import { Link } from "react-router-dom";
 
 export function Navbar() {
-    return (
-        <nav className="text-secondary-900">
-        <ul>
-            Navbar
-        </ul>
+    
+    const [navbar,setnavbar] = useState(false);
+    const [menu,setmenu] = useState(false)
+    
+    const showmenu = () =>{
+        setmenu((prev) => !prev)
+    }
+    const changebg = () =>{
+        if (window.scrollY >= 100){
+            setnavbar(true)
+        } else{
+            setnavbar(false)
+        }
+    }
+    window.addEventListener('scroll',changebg);
+
+        return (
+        <nav className={navbar ? "text-secondary-900 bg-background-950 w-screen fixed duration-700" : "text-secondary-900 bg-transparent w-screen fixed duration-500"}>
```

Comment:
Apply string interpolation and template literals to handle conditional class names more cleanly

```suggestion
        <nav className={`${navbar ? "text-secondary-900 bg-background-950" : "text-secondary-900 bg-transparent"} w-screen fixed duration-700`}>
```
 
Do similar everywhere else

Reply:
okay